.travis.yml

```diff
@@ -0,0 +1,11 @@
+language: python
+python:
+  
+  - "3.6"
+ 
+# command to install dependencies
+install:
+  - pip install -r requirements.txt
+# command to run tests
+script:
+  - pytest
```

Comment:
Point pytest to the folder containing your application code to fix the Travis build
```yml
script:
  - pytest apicode
```

Reply:
@AineKiraboMbabazi 👆 I have updated this. Sorry for the initial info. I was on rush. The first comment applies to coverage